src/sentry/api/endpoints/group_notes.py

```diff
@@ -8,12 +8,11 @@
 from sentry.api.base import DocSection
 from sentry.api.bases.group import GroupEndpoint
 from sentry.api.serializers import serialize
-from sentry.api.serializers.rest_framework.group_notes import NoteSerializer, seperate_resolved_actors
-
-from sentry.api.fields.actor import Actor
+from sentry.api.serializers.rest_framework.group_notes import NoteSerializer
```

Comment:
should `group_notes` --> `notes`?

Reply:
I don't think so, the note serializer there only applies to group notes. At least, I'm not sure it's worth renaming here, since potentially the `NoteSerializer` should be `GroupNoteSerializer`